internal/broker/sessionmode/consts.go

```diff
@@ -0,0 +1,20 @@
+// Package sessionmode defines the session modes supported by the broker.
+package sessionmode
+
+const (
+	// Login is used when the session is for user login.
+	Login = "login"
+	// LoginOld is the old name for the login session, which is now deprecated but still used by authd until all broker
+	// installations are updated.
+	LoginOld = "auth"
+	// ChangePassword is used when the session is for changing the user password.
+	ChangePassword = "change-password"
+	// ChangePasswordOld is the old name for the change-password session, which is now deprecated but still used by authd
+	// until all broker installations are updated.
+	ChangePasswordOld = "passwd"
+)
+
+// IsChangePasswordMode checks if the given mode is a change-password mode.
+func IsChangePasswordMode(mode string) bool {
```

Comment:
I would still had a small (3 test cases: each mode + an invalid one) tests. That way we have some clear documentation about what is supported or not in tests for the exported mode.

Reply:
obsolete

Reply:
Yeah, was just mentioning in case you didn’t want to remove that function :)

Reply:
Sure, I got that! The only reason I wrote that is to make it clear (both to you and me if I look at the PR again) that I consider this thread as done / ready to be resolved. Sorry if it came across as harsh, wasn't meant that way :)